software/4ga-boards.yml

```
@@ -0,0 +1,14 @@
+name: "4ga Boards"
+website_url: "https://4gaboards.com"
+source_code_url: "https://github.com/RARgames/4gaBoards"
+description: "Straightforward realtime kanban boards management for intuitive task tracking. Featuring an elegant dark mode, collapsible todo lists, and multitasking tools to supercharge your team's productivity."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+  - Docker
+  - K8S
```

Changed order to reflect https://github.com/RARgames/4gaBoards?tab=readme-ov-file#deploy and removed K8S since its not present here.
```suggestion
platforms:
  - Docker
  - Nodejs
```

---

It's described here: https://github.com/RARgames/4gaBoards/blob/main/docs/ADDITIONAL_INFO.md#helm-chart

Sorry for the inconvenience, we'll be migrating to new docs shortly to have all information in one place.